src/tests/multihost/alltests/test_misc.py

```diff
@@ -353,3 +353,48 @@ def test_0006_getent_group(self, multihost,
         assert "group-2@example1:*:20002:user-2@example1," \
                "user-4@example1,user-6@example1," \
                "user-8@example1" in cmd.stdout_text
+
+    @pytest.mark.tier1
+    def test_0007_getent_admproxy(self, multihost, backupsssdconf):
+        """
+        :title: 'getent passwd adm@proxy' doesn't
+         return anything when 'cache_first = True' option is used with
```

Comment:
title is too long, some content can be added as
```:description: <more details>```

Reply:
Added description and shortened the title, I hope it looks acceptable now?